### DISCUSSION TITLE
Implement indentation guide lines in tree mode

### DISCUSSION THREAD

**Discussion by nitzcard** [josdejong/svelte-jsoneditor#259] [Ideas]
Is it possible to add in json/tree mode indentation guides? 
Like in vscode: "editor.guides.indentation": true

![image](https://user-images.githubusercontent.com/123297447/232461908-48e41e40-9bea-4238-8b13-507755a5b2db.png)

**Comment by josdejong** [comment]
Thanks for your suggestion @nitzcard. I've moved your idea to the discussions section.

So you mean to implement indentation guide lines in `tree` mode, right? The `text` mode already has indentation guides, but the screenshot you post is from the `jsoneditor` library and not `svelte-jsoneditor`.

If anyone is interested in implementing that please let me know, help would be welcome.

**Reply by nitzcard** [reply]
Yes,I mean for tree mode. Thanks for the quick response. I may make a pr from another user.
